budget_proj/budget_app/models.py

```diff
@@ -34,6 +42,7 @@ class BudgetHistory(models.Model):
     functional_area_name = models.CharField(max_length=255, default='')
     accounting_object_name = models.CharField(max_length=255, default='')
     service_area_code = models.CharField(max_length=32, default='')
+    service_area = models.ForeignKey(LookupCode)
```

Comment:
I think this breaks our importcsv for BudgetHistory.  Now that there's a foreign  key dependency it needs logic to query and find the appropriate LookupCode_id  based on the service_area_code.

I think we have some possible workarounds:
- update BudgetHistory.service_area migration to to query and populate this field. [example](http://stackoverflow.com/questions/31914222/django-new-foreignkey-field-based-on-existing-field)
- Use the normal auto-generated migration, Add the foreign keys locally through ORM (i.e. `manage.py shell`), regenerate Budget History CSVs with a new column containing  LookupCode.id (see `manage.py dumpdata`.  You'll need to  add `null=True` or provide a 1-off value in your migration.
- digging into the docs a little more, it seems adding[ natural keys](https://docs.djangoproject.com/en/1.10/topics/serialization/#topics-serialization-natural-keys) to our model might be easier.